src/qudi/hardware/awg/keysight_M3202A.py

```diff
@@ -152,10 +152,10 @@ def on_activate(self):
             self.awg.close()
             raise Exception('AWG Error: {0} {1}'.format(aouID, ksd1.SD_Error.getErrorMessage(aouID)))
```

Comment:
Are all placed handled where string buffers are used with the keysight DLL? Seems to me like `getErrorMessage` could suffer from the same issue. Basically all DLL calls that return a string should be checked as far as I understand.

Reply:
Interestingly this is not the case: This error only appears because empty string buffers are created by the keysight and filled by `\0`.
The function `getErrorMessage` works with `c_char_p`, so no NULL filled buffers appear. 
All the other hardware calls in the hardware wrapper either return Int-like Status bytes or even proper ints, so this does not appear. A quick test on Hardware, with uploading sequences and playing them did not show any more NULL bytes.

Reply:
The inconsistency in Keysight engineering processes never ceases to amaze me. Ok, thanks. Looks good then.